CMakeLists.txt

```diff
@@ -0,0 +1,75 @@
+#############
```

Comment:
W tym projekcie będę komentował, ogólnie całokształt, a jeżeli chodzi o styl to trzymał się będę tego na którym kiedyś pracowaliśmy ;)

Reply:
Jasne, o to chodzi! Dzięki :) Miło by było jakbyś też skomentował rozwiązania dt. struktur danych. Poszedłem tu dość eksperymentalnie i nie jestem pewien, czy z sensem.